sdk/bootstrap/main.go

```diff
@@ -251,6 +260,78 @@ func renderTemplates(templateMaxBytes int64) {
 	}
 }
 
+func installDCOSCertIntoJRE() {
+	mesosSandbox := os.Getenv("MESOS_SANDBOX")
+	sslDir := filepath.Join(mesosSandbox, ".ssl")
+
+	// Check if .ssl directory is present
+	sslDirExists, err := _isDir(sslDir)
+	if !sslDirExists || err != nil {
+		log.Println("No $MESOS_SANDBOX/.ssl directory found. Cannot install certificate.")
+		log.Fatal(err)
```

Comment:
Also may as well include the `err` on the same line, via a format parameter. see e.g. L210